### ISSUE TITLE
nvidia gtx960, 224x224 image input problem

### ISSUE THREAD

**Issue by ismymajia** [weiliu89/caffe#31]
I use a machine with NVIDIA gtx960 graphics, 2g memory, oom appears when training 300x300 ssd model.

so I modified batch_size = 2, image size = 224, can train, train 60,000 steps and loss = 5.11.

then I test ssd_detect.ipynb with image_resize=224, running this ipynb but can not success localizate which person and fish.  

when test ssd_detect.ipynb with image_resize=300, can success localizate .

**Comment by weiliu89** [comment]
You should change [this](https://github.com/weiliu89/caffe/blob/ssd/examples/ssd/ssd_pascal.py#L253) to 224 as well.

I used examples/ssd/score_ssd_pascal.py and comment out [this](https://github.com/weiliu89/caffe/blob/ssd/examples/ssd/score_ssd_pascal.py#L367) to test the FPS. You could potentially use `build/tools/caffe time ` to test the speed as well.

**Comment by ismymajia** [comment]
after modifing min_dim=224,  mbox_loss = nan appears 
 
I0712 11:00:08.742609 16164 sgd_solver.cpp:106] Iteration 480, lr = 0.001
I0712 11:00:10.707285 16164 solver.cpp:231] Iteration 490, loss = 8.77598
I0712 11:00:10.707331 16164 solver.cpp:247]     Train net output #0: mbox_loss = 7.08969 (* 1 = 7.08969 loss)
I0712 11:00:10.707341 16164 sgd_solver.cpp:106] Iteration 490, lr = 0.001
I0712 11:00:12.655468 16164 solver.cpp:231] Iteration 500, loss = 13.6239
I0712 11:00:12.655514 16164 solver.cpp:247]     Train net output #0: mbox_loss = 9.93708 (* 1 = 9.93708 loss)
I0712 11:00:12.655524 16164 sgd_solver.cpp:106] Iteration 500, lr = 0.001
I0712 11:00:14.615031 16164 solver.cpp:231] Iteration 510, loss = 9.13577e+18
I0712 11:00:14.615072 16164 solver.cpp:247]     Train net output #0: mbox_loss = 9.13577e+19 (* 1 = 9.13577e+19 loss)
I0712 11:00:14.615083 16164 sgd_solver.cpp:106] Iteration 510, lr = 0.001
I0712 11:00:16.561933 16164 solver.cpp:231] Iteration 520, loss = nan
I0712 11:00:16.561977 16164 solver.cpp:247]     Train net output #0: mbox_loss = nan (* 1 = nan loss)
I0712 11:00:16.561987 16164 sgd_solver.cpp:106] Iteration 520, lr = 0.001
I0712 11:00:18.506747 16164 solver.cpp:231] Iteration 530, loss = nan
I0712 11:00:18.506789 16164 solver.cpp:247]     Train net output #0: mbox_loss = nan (* 1 = nan loss)
I0712 11:00:18.506799 16164 sgd_solver.cpp:106] Iteration 530, lr = 0.001
I0712 11:00:20.451352 16164 solver.cpp:231] Iteration 540, loss = nan
I0712 11:00:20.451396 16164 solver.cpp:247]     Train net output #0: mbox_loss = nan (* 1 = nan loss)
I0712 11:00:20.451406 16164 sgd_solver.cpp:106] Iteration 540, lr = 0.001
I0712 11:00:22.389009 16164 solver.cpp:231] Iteration 550, loss = nan
I0712 11:00:22.389052 16164 solver.cpp:247]     Train net output #0: mbox_loss = nan (* 1 = nan loss)
I0712 11:00:22.389061 16164 sgd_solver.cpp:106] Iteration 550, lr = 0.001
I0712 11:00:24.331504 16164 solver.cpp:231] Iteration 560, loss = nan
I0712 11:00:24.331545 16164 solver.cpp:247]     Train net output #0: mbox_loss = nan (* 1 = nan loss)
I0712 11:00:24.331555 16164 sgd_solver.cpp:106] Iteration 560, lr = 0.001
I0712 11:00:26.274636 16164 solver.cpp:231] Iteration 570, loss = nan
I0712 11:00:26.274678 16164 solver.cpp:247]     Train net output #0: mbox_loss = nan (* 1 = nan loss)
I0712 11:00:26.274688 16164 sgd_solver.cpp:106] Iteration 570, lr = 0.001
I0712 11:00:28.209434 16164 solver.cpp:231] Iteration 580, loss = nan
I0712 11:00:28.209477 16164 solver.cpp:247]     Train net output #0: mbox_loss = nan (* 1 = nan loss)
I0712 11:00:28.209487 16164 sgd_solver.cpp:106] Iteration 580, lr = 0.001
I0712 11:00:30.145925 16164 solver.cpp:231] Iteration 590, loss = nan
I0712 11:00:30.145968 16164 solver.cpp:247]     Train net output #0: mbox_loss = nan (* 1 = nan loss)

**Comment by weiliu89** [comment]
You could try to use smaller learning rate to train for a couple hundred iterations (warm start), stop and resume with 0.001.

**Comment by KylanRace** [comment]
@ismymajia , I am trying to do the same thing by reducing my images down to 224x224, though i noticed the loss is very high for your VGG model (when run with caffe time). Did you experience the same thing / do you have a model that you successfully trained?
